spec/controllers/admin/enterprises_controller_spec.rb

```diff
@@ -319,6 +319,19 @@
           expect(voucher_a.reload.deleted?).to be(true)
         end
 
+        context "when desactivating the only existing voucher" do
+          it "deactivates the voucher" do
+            voucher_b.really_destroy!
```

Comment:
I think we don't need to create `voucher_b` in the first place do we? This line could get confused with the rest of the test conditions.
I think the other tests would still work if `voucher_b` was simply converted to an optional `let`.

Reply:
Good find !